pyserverforindy/grpc_server/anoncreds_service.py

```diff
@@ -2,7 +2,11 @@
 import os
 import sys
 
-from indy import anoncreds as indy_anoncreds
+from indy import anoncreds as indy_anoncreds,IndyError
+from indy.error import ErrorCode
+from grpc import StatusCode
+
+
 
 ROOT_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
```

Comment:
No major changes in annoncreds_service. just InnyError handler for future changes